pkg/collector/tcp/tcp.go

```diff
@@ -35,6 +37,7 @@ type Collector struct {
 	segmentsReceivedTotal      *prometheus.Desc
 	segmentsRetransmittedTotal *prometheus.Desc
 	segmentsSentTotal          *prometheus.Desc
+	TCPState                   *prometheus.Desc
```

Comment:
```suggestion
	tcpState                   *prometheus.Desc
```

alle files should be keep private

Reply:
ok replaced TCPState with connectionsStateCount